code/modules/events/gimmick/bigfart.dm

```diff
@@ -157,9 +158,7 @@
 		boutput(H, "<span class='notification'>[changer ? "We" : "You"] hear an otherworldly force let out a short, disappointed cluck at [changer ? "our" : "your"] lack of an arse.</span>")
 	H.visible_message("<span class='alert'>[is_bot ? "Oily chunks of twisted shrapnel" : "Wadded hunks of blood and gore"] burst out of where <b>[H]</b>'s [magical ? "arse" : "ass"] used to be!</span>",\
 	"<span class='alert'>[nobutt_phrase[assmagic]]</span>")
-	if(magical)
-		H.changeStatus("weakened", 1 DECI SECOND)
-	else
+	if(!magical)
 		H.changeStatus("weakened", 3 SECONDS)
 	H.force_laydown_standup()
```

Comment:
probably shouldn't be calling `force_laydown_standup()` if you aren't using it to apply weakness